packages/agents/sdk-wrapper/src/sdkPool.ts

```diff
@@ -39,10 +39,14 @@ export class SdkPool extends SdkShared {
     amount: BigNumberish,
     options?: Options,
   ): Promise<BigNumber> {
-    const _options = options ?? {
-      chains: this.config.chains,
-      signerAddress: this.config.signerAddress,
-    };
+    const originProviderUrl = this.config.chains[domainId]?.providers;
+
+    const _options = options ?? {};
```

Comment:
We still need to pass chains and signerAddress from config

Reply:
So these specific functions dont really require chains and signerAddress internally.

Reply:
swap() will need signerAddress, for example